src/xdocs/checks/blocks/avoidnestedblocks.xml

```diff
@@ -112,23 +113,23 @@ public void foo() {
         </source>
         <p id="Example2-code">Example:</p>
         <source>
-public void foo() {
-  int myInteger = 0;
-  {                      // violation
-    myInteger = 2;
-  }
-  System.out.println(&quot;myInteger = &quot; + myInteger);
+public class Example2 {
+  public void foo() {
+    int myInteger = 0;
+    {                      // violation
+      myInteger = 2;
+    }
+    System.out.println(&quot;myInteger = &quot; + myInteger);
 
-  switch (a)
-  {
-    case 1:
-      {                    // OK
+    switch (a) {
+      case 1: {                    // OK
         System.out.println(&quot;Case 1&quot;);
         break;
       }
-    case 2:
-      System.out.println(&quot;Case 2&quot;);     // OK
-      break;
+      case 2:
+        System.out.println(&quot;Case 2&quot;);     // OK
```

Comment:
Please remove OK comments, we allowing them only if it has extra text to explain why it is ok.
You have few in your examples.

Reply:
> Please remove OK comments, we allowing them only if it has extra text to explain why it is ok. You have few in your examples.
Done.